scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -76,6 +76,8 @@ SG_ STACK_WATERMARK_ABOVE_THRESHOLD_TASKCANRX : 4|1@1+ (1,0) [0|1] "" LOGGER
 SG_ STACK_WATERMARK_ABOVE_THRESHOLD_TASKCANTX : 5|1@1+ (1,0) [0|1] "" LOGGER
 SG_ WATCHDOG_TIMEOUT : 6|1@1+ (1,0) [0|1] "" LOGGER
 SG_ BSPD_FAULT : 7|1@1+ (1,0) [0|1] "" DIM
+SG_ LEFT_WHEEL_SPEED_OUT_OF_RANGE: 8|1@1+ (1,0) [0|1] "" LOGGER
```

Comment:
These should be 2 bits (`8|2@1+`) and have the following enum

https://github.com/UBCFormulaElectric/Consolidated-Firmware/blob/a17be8a4b86bdd5ae4463dc64d81b4737a4c7c45/scripts/codegen/CAN/App_CanMsgs.dbc#L268

Reply:
done